content/changelog/2024-01-26-console.md

```diff
@@ -0,0 +1,55 @@
+---
+description: Vercel integration updates, fixes & updates, and more
+---
+
+### New default Postgres version
+
+The default version for newly created Neon projects is now Postgres 16. Neon continues to support Postgres 14 and 15.
+
+### Vercel integration updates
```

Comment:
@danieltprice I like how the previous header: "New default Postgres version" gives you the TL;DR right in the header, making the whole page more scannable.

Could we do the same for all these "big update" sections that we put at the top of each post? 

So in the case of this one, maybe:

```suggestion
### Configurable ENV vars for Vercel Integration
```

Reply:
Changed!

Reply:
We'll do this going forward